### DISCUSSION TITLE
Differences between MAS version and Gumroad?

### DISCUSSION THREAD

**Discussion by Kookaburra2027** [p0deje/Maccy#981] [Q&A] [answered]
Hello, I have been unable to find a definitive guide about what are the differences between the Mac App Store version and the Gumroad version of Maccy. Can anyone please point me in the right direction?

I am specifically looking for are there any:
-security differences (sandboxing)
-network permissions (I am very keen on having no network connections where possible).

Thanks in advance

**Comment by p0deje** [comment, marked as answer]
The only difference between the two are:
- MAS asks for a review;
- MAS uses a built-in auto-update mechanism so Sparkle is not included;
- MAS has a different distribution account assigned.

The rest is the same. Regarding your question, both are using sandboxes and don't require network permissions.

**Reply by Kookaburra2027** [reply]
Thanks for the clarification - much appreciated.

Just to confirm, so there is no network access other than updates in either version?

**Reply by p0deje** [reply]
That's correct.

**Reply by Kookaburra2027** [reply]
Thanks again, very grateful for the clarification :-)
